src/avram/database.cr

```diff
@@ -38,6 +38,12 @@ abstract class Avram::Database
     new.delete
   end
 
+  @@database_info : DatabaseInfo?
+
+  def self.database_info : DatabaseInfo
+    @@database_info ||= DatabaseInfo.load(self)
```

Comment:
is `self` in this context the concrete class of `Avram::Database`?

Reply:
Yes. I had everything in `DatabaseInfo.load` in this method but moved it in effort to clean up